### ISSUE TITLE
Chlorine and Arcadia

### ISSUE THREAD

**Issue by calmaty** [mauricioszabo/atom-chlorine#192]
Hey i don't know if this is the right place to write.

I have been using chlorine to connect to arcadia: https://github.com/arcadia-unity/Arcadia

and recently when i updated to version 0.7.2 i stopped being able to do that.

I previously used Connect Clojure Socket Repl but that seemed to have been removed after 0.5.4

And Connect Socket Repl won't work for me.

Right now i have reverted to using 0.5.4 but i would love to be able to use the newer versions in the future if possible.

**Comment by mauricioszabo** [comment]
Yes, it's the right place. I'll look at it :)

Probably a consequence of this: https://github.com/arcadia-unity/Arcadia/issues/375.

**Comment by mauricioszabo** [comment]
@calmaty Just some more info:

Are you using Socket REPL to connect?
Are you on which OS?
Also, do you see some error, or it simply does nothing at all?

**Comment by mauricioszabo** [comment]
I think it's fixed on 0.7.3. Can you check it, please? On my machine, I could somewhat reproduce the problem by toggling some "wait" code that I have on the connection...

It it still happens, please feel free to reopen this issue :)

**Comment by calmaty** [comment]
just tried to connect, it still dosn't work on my machine. i run windows and i get this error:

```
System.IO.FileNotFoundException: Could not locate orchard.info.cljc.dll or orchard/info.cljc on load path.
* clojure/core/load-libfn
* clojure/core/load-lib
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/load-libs
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/require
* clojure.lang.RestFn
* user/eval
  clojure.lang.Compiler.eval (object)
* clojure/core/eval
* clojure.lang.Var
* Arcadia.NRepl+EvalFn Cardlike/Assets/Arcadia/Editor/NRepl.cs (203:0)System.IO.FileNotFoundException: Could not locate orchard.xref.cljc.dll or orchard/xref.cljc on load path.
* clojure/core/load-libfn
* clojure/core/load-lib
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/load-libs
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/require
* clojure.lang.RestFn
* user/eval
  clojure.lang.Compiler.eval (object)
* clojure/core/eval
* clojure.lang.Var
* Arcadia.NRepl+EvalFn Cardlike/Assets/Arcadia/Editor/NRepl.cs (203:0)System.IO.FileNotFoundException: Could not locate orchard.clojuredocs.cljc.dll or orchard/clojuredocs.cljc on load path.
* clojure/core/load-libfn
* clojure/core/load-lib
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/load-libs
* clojure.lang.RestFn
* clojure/core/apply
* clojure/core/require
* clojure.lang.RestFn
* user/eval
  clojure.lang.Compiler.eval (object)
* clojure/core/eval
* clojure.lang.Var
* Arcadia.NRepl+EvalFn Cardlike/Assets/Arcadia/Editor/NRepl.cs (203:0)

```

**Comment by mauricioszabo** [comment]
@calmaty Chlorine _prefers_ Socket REPL. I believe you're trying to connect to a nREPL port - try to connect to port `37220`.

(Just to be sure, it's still a bug - I just didn't implement nREPL specifically for Arcadia, so that can be the problem)

**Comment by calmaty** [comment]
I can't connect to 37220

`Connection refused. Ensure that you have a Socket REPL started on this host/port`

**Comment by mauricioszabo** [comment]
Hello there. I just published release 0.7.5, it probably solved these errors on the console.

Now, I must be honest - I have no idea how are you connecting to Arcadia on Chlorine 0.5.x. The reason you're seeing this error happening on the console is probably because you were connecting in a nREPL port, and that's not supported on 0.5.4...

Well, I've installed Arcadia on my Linux machine, and it opened both a nrepl and a socket REPL ports. With 0.7.5, I'm being able to connect and evaluate code on both. Please, let me know if it's still not working for you, feel free to re-open this issue. Also, if you could send me your operating system, Arcadia and Atom version, I believe I could debug it further :)

**Comment by calmaty** [comment]
Everything works now.

Things where actually fixed in 0.7.4 but i misinterpreted the error message (which was removed in 0.7.5)

Thanks for the time and patience.
